README.md

```diff
@@ -130,23 +128,20 @@ This version is slower (about 5x) so it is offered as an opt-in.
 To use the dedupe version with Node.js, Browserify, or webpack:
 
 ```js
-var classNames = require('classnames/dedupe');
+import classNames from 'classnames/dedupe.js';
```

Comment:
One of the 'downsides' of using the ES module syntax is that it is [mandatory to provide a file extension](https://nodejs.org/api/esm.html#esm_mandatory_file_extensions) for files.

Reply:
This can be worked around by adding [entry points](https://nodejs.org/api/packages.html#packages_package_entry_points) in `package.json`.

```js
{
  // …
  "exports": {
    ".": "./index.js",
    "./bind": "./bind.js",
    "./dedupe": "./dedupe.js"
  },
  // …
}
```

Although personally I prefer using named exports.

```js
import { classNames, bind, dedupe } from 'classnames';
```

Reply:
Nice, I'll add the entry points to the package. We'll have to discuss the named exports, they could be added as a feature later also.

Reply:
Ok, I've added a commit for the entry points. I'm resolving this one.